docs/pages/database-access/guides/azure-sql-server-ad.mdx

```diff
@@ -3,7 +3,7 @@ title: Database Access with SQL Server on Azure
 description: How to configure Teleport database access with Azure SQL Server using Azure Active Directory authentication.
 ---
 
-(!docs/pages/includes/database-access/db-introduction.mdx  dbType="Azure SQL Server" dbConfigure="Azure SQL Server using Azure Active Directory authentication" dbName="Azure SQL Server" !)
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="Azure SQL Server" dbConfigure="using Azure Active Directory authentication"!)
```

Comment:
nit: Azure Active Directory is renamed to Microsoft Entra ID
https://learn.microsoft.com/en-us/azure/azure-sql/database/authentication-aad-overview?view=azuresql